docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/OneVersusAll.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.MulticlassClassification
+{
+    public static class OneVersusAll
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of data examples.
+            var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
```

Comment:
>GenerateRandomMulticlassClassificationExamples [](start = 40, length = 46)

is it possible to use something like GenerateRandomDataPoints that's used in binary classification?
[https://github.com/dotnet/machinelearning/blob/0ce5618516a069f173467020abcadb99a0648349/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FastTree.cs#L68](https://github.com/dotnet/machinelearning/blob/0ce5618516a069f173467020abcadb99a0648349/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FastTree.cs#L68)

Reply:
This method generates data points for multi class, with 4 labels. Except for this it does do it similarly to the GenerateRandomDataPoints. Not sure what other similarity you would want?

---
In reply to: [271461899](https://github.com/dotnet/machinelearning/pull/3159#discussion_r271461899) [](ancestors = 271461899)

Reply:
this is actually done, thx

---
In reply to: [271497239](https://github.com/dotnet/machinelearning/pull/3159#discussion_r271497239) [](ancestors = 271497239,271461899)